apostrophecms/apostrophe
# Pro 3523 prefill

## Summary

Add new autocomplete results UI

## What are the specific steps to test this change?

1. Create a relationship schema fields
2. Check the new UI
3. Set the relationship field `fields` with some attributes, and check that this is displayed on the result list
   ```
   fields: [ 'slug', 'xxx' ]
   ```
4. Set `browse: false` on the relationship and verify that the `Browse` button as disappeared
5. You can also change the following piece options or relationship field property
   *  `relationshipSuggestionLabel` | `suggestionLabel`
   * `relationshipSuggestionHelp` | `suggestionHelp`
   * `relationshipSuggestionLimit` | `suggestionLimit`
   * `relationshipSuggestionIcon` | `suggestionIcon`
   * `relationshipSuggestionFields` | `suggestionFields`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
